Waveformer.py

```diff
@@ -62,11 +107,13 @@
     else:
         query = torch.zeros(1, len(TARGETS))
         for t in args.targets:
-            query[0, TARGETS.index(t)] = 1.
+            query[0, TARGETS.index(t)] = 1.0
 
-    with torch.no_grad():
-        output = model(mixture, query)
+    with torch.inference_mode():
```

Comment:
inference mode is recommended https://discuss.pytorch.org/t/pytorch-torch-no-grad-vs-torch-inference-mode/134099/2?u=timgianitsos